python_frank_energie/models.py

```
@@ -3936,12 +3943,14 @@ class Session:
     @staticmethod
     def from_dict(payload: dict[str, object]) -> SmartBatterySessions.Session:
         """Parse the sessions payload from the SmartBatterySessions query result."""
-        _LOGGER.debug("🔁 Parsing SmartBatterySessions.Session response: %s", payload)
+        _LOGGER.debug("Parsing SmartBatterySessions.Session response: %s", payload)
 
         try:
             return SmartBatterySessions.Session(
```

Pylint (4.0.6)

[error] Class 'SmartBatterySessions' has no 'Session' member

(E1101)